manifests/oss/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resources:
+- ../base
+- ../test-resources/00-namespace.yaml
```

Comment:
While we're here, it would be good if we can rename the `test-resources` directory as it is not test resources, but the namespaces and resource-group-controller that are needed for the multi-repo mode.

Reply:
Agreed that the `test-resources` directory is a misnomer. Would you mind if I update that in a separate change? I'll have to update the naming in a few places so I'd like to isolate that as a separate change

Reply:
SG

Reply:
https://github.com/GoogleContainerTools/kpt-config-sync/pull/256